### ISSUE TITLE
Q: plans to have a conda package with conda-forge

### ISSUE THREAD

**Issue by thorade** [linkchecker/linkchecker#186]
Just a question, low priority, and after #4 and #40:
It would be awesome to at least consider adding a recipe to conda-forge.
https://anaconda.org/conda-forge
https://conda-forge.org/
Some anaconda users prefer conda over pip.

**Comment by anarcat** [comment]
what is conda-forge and why should we care? could you expand? :)

**Comment by thorade** [comment]
Anaconda is one popular Python distribution, and they have their own package manager conda, which has some advantages over pip, it is especially easier to package binaries.

There are official packages for many Python libraries, like scipy, numpy etc, and also many smaller libraries. Official means: packaged and tested by the makers of Anaconda.

Everybody can create "channels" and distribute new conda packages, but one of the most popular channels is conda-forge. They use recipes, which are some kind of build instruction, to reproducibly create conda packages from official source (e.g. your repo here).

So, you should only care if you consider if you want to help those Anaconda users who prefer conda over pip.

**Comment by anarcat** [comment]
well, i'd be happy to see linkchecker packaged elsewhere, but i'm not sure we should be handling this ourselves...?

**Comment by thorade** [comment]
Once there is a new PyPi release, I can try to build a recipe for conda-forge. I have never done that, but I heard it is doable.

**Comment by anarcat** [comment]
@thorade that's great news, but i encourage you not to wait for PyPI. There are regular releases here on github you can build on.

**Comment by phaer** [comment]
@thorade Just a heads-up, the most recent release (from January this year) is on PyPi, so as I understand this thread all should be ready for a conda-forge recipe, if you are still motivated :)

**Comment by cjmayo** [comment]
Let's say that the project publishes on GitHub and PyPI - requests for packaging in other formats should go to the organisations managing those repositories. If there are issues with packaging that the project could make easier do raise.

There would also be an issue for us with:
"Your recipe will automatically be built on Windows, Linux, and OSX" [1]
In that we only have testing on Linux at the moment and especially Windows where there seem to be problems #494.

For Linux users they may well have a package from their distribution - and currently the advice for Windows is to use WSL which gives them access to the Ubuntu package.

[1] https://conda-forge.org/#contribute
